repository/repository/root.json

```diff
@@ -2,139 +2,163 @@
 	"signed": {
 		"_type": "root",
 		"spec_version": "1.0",
-		"version": 8,
-		"expires": "2024-03-26T04:38:55Z",
+		"version": 9,
+		"expires": "2024-09-12T06:53:10Z",
 		"keys": {
-			"25a0eb450fd3ee2bd79218c963dce3f1cc6118badf251bf149f0bd07d5cabe99": {
-				"keytype": "ecdsa-sha2-nistp256",
+			"1e1d65ce98b10addad4764febf7dda2d0436b3d3a3893579c0dddaea20e54849": {
```

Comment:
This is Santiago's key. The diff looks larger than it is as the order has been changed with the new key ids, so both public key (PEM encoded) and the key id is different.